### DISCUSSION TITLE
#87: aspect-ratio:1 seems not to be working

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#220] [Q&A]
I optimized my solution in target 87 and I understand not, why aspect-ration is not working, when I put my code the tool https://codepen.io/alexzaworski/full/vwmNKr 

It makes  no difference if I put `aspect-ratio:1`in `*`ore `*>*` 

`<p><p><style>*{background:var(--b,#F3AC3C);color:1A4341;aspect-ratio:1}p{width:50;--m:-165 100;--:21q 127q}p+p{width:70;--b:#F3AC3C;--m:165 100;--:42q 53q,-42q -53q}*>*{width:100;--b:#1A4341;margin:var(--m,230 65);border-radius:10px;box-shadow:var(--,180q 74q`

Thanks for give me an idea what happens here!

**Comment by gunnarbittersmann** [comment]
According to [Can I Use/MDN](https://caniuse.com/mdn-css_properties_aspect-ratio) `aspect-ratio` has landed in Chromium 88. Maybe the Chromium on the server is older?

**Reply by chinchang** [reply]
Yep. it's pretty old. I'll update it soon.
